test/hotspot/jtreg/vmTestbase/nsk/jdb/kill/kill001/kill001a.java

```diff
@@ -152,6 +153,17 @@ void methodForException() {
     }
 
     public void run() {
+        boolean vthreadMode = "Virtual".equals(System.getProperty("test.thread.factory"));
+        if (vthreadMode) {
+            // JVMTI StopThread is only supported for mounted virtual threads. We need to
+            // pin the virtual threads so they remain mounted.
+            VThreadPinner.runPinned(() -> run1());
+        } else {
+            run1();
+        }
+    }
+
+    public void run1() {
```

Comment:
It might be a bit clear to rename run1 to runPinned (avoids run and run1).

Reply:
It also needs to support test runs with platforms threads. See how `run()` checks if it is running in virtual thread mode or not. Either way it calls `run()`, but in the virtual thread case it does so using VThreadPinner.